pandevice/userid.py

```diff
@@ -240,7 +253,7 @@ def register(self, ip, tags):
                 member.text = tag
         self.send(root)
 
-    def unregister(self, ip, tags):
+    def unregister(self, ip, tags, nonexistant=None):
```

Comment:
The reason I chose to do it this way is because different ip address will have different invalid tags. Thus if I chose to do the processing of nonexistant tags outside of unregister, I would most likely have to create a different unregister call for each individual ip address.
I'm not sure which way would be preferable.